app/components/SendModal/index.js

```diff
@@ -192,13 +252,25 @@ class SendModal extends Component {
           <div className="send__to">
             <div className="send__label">{t('sendToLabel')}</div>
             <div className="send__input" key="send-input">
+              {hip2Input && <span className="send__prefix">{to ? (
+                <img src={LockSVG} />
+              ) : '@'}</span>}
               <input
                 type="text"
-                placeholder={t('recipientAddress')}
+                className={hip2Input ? (to ? 'send__input-hip2-success' : 'send__input-hip2') : null}
+                placeholder={hip2Input ? t('recipientHip2Address') : t('recipientAddress')}
                 onChange={this.updateToAddress}
-                value={to}
+                onKeyDown={this.updateHip2}
+                spellCheck="false"
+                value={hip2Input ? hip2To : to}
               />
             </div>
+            { hip2Input && (
+              <React.Fragment>
+                <Alert type="error" message={hip2Error} />
```

Comment:
This error stays as the domain is typed out. It's kinda hard to make out if the domain I've just typed really doesn't work, or if it isn't resolved yet. Can we clear the error soon as input changes (`onChange`)?
Even better, some kind of a tiny loading spinner if possible.